src/wglmakie.js

```diff
@@ -630,8 +630,10 @@ const WGLMakie = (function () {
         });
 
         renderer.setClearColor("#ffffff");
-        renderer.setPixelRatio(1);
-
+        renderer.setPixelRatio(window.devicePixelRatio);
+        
+        renderer.setSize(window.innerWidth, window.innerHeight + 20);
```

Comment:
+ 20? What's that doing?

Reply:
The basic idea was to increase the dpi and then scale it down

+ 20 because when I did scatter(1:4), the upper part got a bit cut off

Reply:
That sounds weird :D We should find the cause of the cut off and fix it!
Where did it happen?

Reply:
You're right
Its really not the best solution
It happens when you try `scatter` 
Oddly, the problem still persists

I feel that the best solution would be to set a default size, and then make the size configurable by the user

Reply:
I mean, where (browser, jupyter vscode) do you display this when it gets cut off...Do you have a screenshot?

Reply:
![Screenshot from 2021-05-15 00-29-10](https://user-images.githubusercontent.com/51269425/118316643-b4123e80-b514-11eb-87b5-9b6d637488c5.png)
Pluto

Reply:
Also, I feel that a simple scatter plot, should occupy that amount of space.....